cocoa/src/toga_cocoa/app.py

```diff
@@ -244,9 +268,20 @@ def create(self):
                 section=10,
                 order=60,
             ),
+            # ---- Edit menu ----------------------------------
+            toga.Command(
+                lambda _, **kwargs: self.interface.current_window._impl.native.miniaturize(
+                    None
+                )
+                if self.interface.current_window
+                else None,
+                "Minimize",
+                shortcut=toga.Key.MOD_1 + "m",
+                group=toga.Group.WINDOW,
+            ),
```

Comment:
While the "Cmd + m" shortcut is idiomatic, having a visible "Minimize" in the Edit menu doesn't seem to be. What is your point of reference here?

Reply:
That's a copy-and-paste error in the comment. The code puts it in the Window group.